pikaur/config.py

```diff
@@ -56,6 +56,12 @@
             'type': 'int',
             'default': '9',
         },
+    },
+    'misc': {
```

Comment:
mb `ui` instead of `misc`?

Reply:
That's probably a more reasonable name, yeah. I blanked.